src/pdn/src/straps.cpp

```diff
@@ -293,8 +293,8 @@ void Straps::makeStraps(int x_start,
         // no portion of the strap is inside the limit
         return;
       }
-      if (group_pos > pos_end) {
-        // strap center is outside of alotted area
+      if (strap_end > pos_end) {
+        // strap end is outside of alotted area
```

Comment:
Is the prior check
```
if (strap_start >= pos_end) {
        // no portion of the strap is inside the limit
        return;
      }
```
now redundant?  I assume this check would always fail in that case

Reply:
you are correct